test/pagination-query.ts

```diff
@@ -0,0 +1,205 @@
+import { url } from "../src/setup-server";
+import { expect } from "chai";
+import { prisma } from "../src/setup-db";
+import { userSeed } from "../src/user-seed";
+import { defaultLimit } from "../src/graphql/schema";
+import { generateToken } from "../src/graphql/helpers/login-handlers";
+import axios from "axios";
+
+const getManyUsersPagination = `#graphql 
+  query GetManyUsers ($offset: Int, $limit: Int) {
+    getManyUsers(offset: $offset, limit: $limit) {
+      totalCount
+      hasMoreUsers
+      users {
+        id
+        name
+        email
+      }
+    }
+  }`;
+
+async function serverRequest(url: string, token: string, limit?: number, offset?: number) {
```

Comment:
```suggestion
async function serverRequest(url: string, token: string, params: { limit?: number; offset?: number }) {
```
ou isso ou { url: string; token: string; limit?: ... }
para ler o uso dessa `serverRequest` não está muito legível pra saber o que é o quê, nesse sentido vale transformar os parametros num objeto quando a função já tem 3 ou mais parametros, pra ficar nomeado bonitinho na hora de usar
```
foo({
  param1: 'bar',
  param2: 'etc',
...
});